### PR TITLE
Add dotnet publish and signing binaries for oryx build server

### PR DESCRIPTION
**Pull request by william-msft**
- [ ] The purpose of this PR is explained in this message or in an issue. If an issue please include a reference as https://devdiv.visualstudio.com/DevDiv/_boards/board/t/Oryx/Stories/?workitem=1477639
- [ ] Tests are included and/or updated for code changes.
- [ ] Proper license headers are included in each file.

### CODE REVIEW

File: vsts/pipelines/ci.yml [hunk 1]
```diff
@@ -76,32 +76,6 @@ stages:
         displayName: 'Set release tag name as environment variable'
       - template: templates/_signBinary.yml
 
-    #- job: Job_SignBinariesBuildServer
```
Comment (daniv-msft) [inline vsts/pipelines/ci.yml:79]: Could you please clarify why you removed this from the pipeline? I would have expected you would have needed it.

Comment (william-msft) [reply vsts/pipelines/ci.yml:79]: Yes, this code is basically referencing _signBinaryBuildServer.yml: https://github.com/microsoft/Oryx/pull/1268/files#diff-58841cda045384dd117e38a9271168bc0332c2776a5ca7330531275083371decL103

Which has the essential code from _signBinaryBuildServer.yml in the _signBinary.yml.

Comment (william-msft) [reply vsts/pipelines/ci.yml:79]: - Signing binaries
- Archiving binaries
- Copying binaries
Are all in the same pipeline job with  _signBinary.yml

Comment (daniv-msft) [reply vsts/pipelines/ci.yml:79]: OK thanks

**Review by daniv-msft — APPROVED** [review]

**Pull request merged**